src/roswire/ros2/msg.py

```diff
@@ -2,73 +2,74 @@
 __all__ = ("ROS2Field")
 
 import re
-from typing import Any, Dict, Optional
+from typing import Any, Dict, List, Optional
 
 import attr
 import dockerblade
 
 from ..common import Field
-from ..common.msg import MsgFormat, R_COMMENT
+from ..common.msg import Constant, MsgFormat, R_BLANK, R_COMMENT
+from ..exceptions import ParsingError
 
 
 @attr.s(frozen=True, str=False, slots=True, auto_attribs=True)
 class ROS2Field(Field):
     R_DEFAULT_VALUE = r"[^#]*"
-    R_FIELD = re.compile(f"^\s*({Field.R_TYPE})(\s+)({Field.R_NAME})"
-                         f"\s+({R_DEFAULT_VALUE})\s*"
-                         f"{R_COMMENT}$")
+    R_FIELD = re.compile(f"^\s*(?P<type>{Field.R_TYPE})"
+                         f"\s+(?P<name>{Field.R_NAME})(?:\s+)?"
+                         f"(?P<val>{R_DEFAULT_VALUE}){R_COMMENT}")
 
-    default_val: Optional[str]
```

Comment:
Is this a safe change? Are there clients that expect Field to have a default_val attribute?

Reply:
Not sure what you mean. It though this was the culmination of what the changes has been about - to allow fields to have a default_val.

Reply:
Suggest rename to `default_value`.

Reply:
It looks like this hasn't been changed yet?

Reply:
I'm very confused about this. If I go to the latest version of the file I see that it is `default_value`. It just doesn't appear in this comment,